src/com/owncloud/android/lib/resources/status/GetRemoteCapabilitiesOperation.java

```diff
@@ -245,8 +245,10 @@ protected RemoteOperationResult run(OwnCloudClient client) {
                                 capability.setFilesUndelete(CapabilityBooleanType.fromBooleanValue(
                                         respFiles.getBoolean(PROPERTY_UNDELETE)));
                             }
-                            capability.setFilesVersioning(CapabilityBooleanType.fromBooleanValue(
-                                    respFiles.getBoolean(PROPERTY_VERSIONING)));
+                            if (respFiles.has(PROPERTY_VERSIONING)) {
```

Comment:
The code looks good to me but could you update the copyright year at the top of this class?